core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,96 @@
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">Collection > {{collection_title}}</span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div ng-controller="CollectionPlayer">
+    <div ng-if="collection" class="oppia-collection-player-tiles-container">
+      <h2 ng-if="!hasFinishedCollection()" class="oppia-page-heading">
+        <span ng-if="!hasStartedCollection()">Begin <[collection.title]>:</span>
+        <span ng-if="hasStartedCollection()">Continue <[collection.title]>:</span>
+      </h2>
+      <h2 ng-if="hasFinishedCollection()" class="oppia-page-heading">
+        <span>You have finished the collection! Feel free to replay any explorations below.</span>
+      </h2>
+
+      {% if is_logged_in %}
+        <div ng-if="getNextRecommendedCollectionNodeCount() > 0">
+          <span class="oppia-collection-player-small-text">
+            Next exploration<span ng-if="getNextRecommendedCollectionNodeCount() > 1">s</span> to play
+          </span>
+          <collection-node-list-directive collection-id="collectionId" collection-nodes="getNextRecommendedCollectionNodes()" />
+        </div>
+
+        <br ng-if="getCompletedExplorationNodeCount() > 0">
+
+        <div ng-if="getCompletedExplorationNodeCount() > 0">
+          <span class="oppia-collection-player-small-text">
+            Completed exploration<span ng-if="getCompletedExplorationNodeCount() > 1">s</span>
+          </span>
+          <collection-node-list-directive collection-id="collectionId" collection-nodes="getCompletedExplorationNodes()" />
+        </div>
+
+        <br ng-if="showingAllExplorations && (getNonRecommendedCollectionNodeCount() > 0)">
+
+        <div ng-if="showingAllExplorations && (getNonRecommendedCollectionNodeCount() > 0)">
+          <span class="oppia-collection-player-small-text">
+            Future exploration<span ng-if="getNonRecommendedCollectionNodeCount() > 1">s</span> in this collection
+          </span>
+          <collection-node-list-directive collection-id="collectionId" collection-nodes="getNonRecommendedCollectionNodes()" />
+        </div>
+
+        <br ng-if="getNonRecommendedCollectionNodeCount() > 0">
+
+        <a class="oppia-collection-player-small-text" ng-if="getNonRecommendedCollectionNodeCount() > 0" ng-click="toggleShowAllExplorations()">
+          <span ng-if="!showingAllExplorations"><span class="fa fa-chevron-down"></span> Show future explorations</span>
```

Comment:
I changed it to 'upcoming' per your earlier suggestion. I had it as 'all' before, but it doesn't seem to fit here because it's not actually revealing a list containing all of the explorations, it's only showing the upcoming ones. Let me know if you still think it should say 'all' (I'm not particular either way; if 'all' makes more sense then I'm fine with it).